src/styles/apps-menu.styl

```diff
@@ -33,3 +32,8 @@
     +desktop()
         width 2.625rem
         height 2.625rem
+
+.apps-menu-grid-item-text
```

Comment:
you should use "makeStyles" (with or without "classes") to override without having to use important, it's the recommande way for Mui component